rakefile.netstd.rb

```diff
@@ -12,8 +12,9 @@
 solution        = "FakeItEasy.netstd.sln"
 assembly_info   = "src/CommonAssemblyInfo.cs"
 version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
-version_suffix  = ENV["VERSION_SUFFIX"]
-version_suffix  = version_suffix.to_s.empty? ? "-adhoc" : version_suffix
+#version_suffix  = ENV["VERSION_SUFFIX"]
+#version_suffix  = version_suffix.to_s.empty? ? "-adhoc" : version_suffix
+version_suffix  = "-netstd" #temporary while we have two build scripts

```

Comment:
👍 I'm content too 😉 

The way I see it is that the only things we are currently pushing to nuget.org are the regular packages, and things won't change in any way with their versioning. The netstd packages will always be suffixed -netstd and live only in the appveyor feed. At the point we are ready to push out netstd beta/RC to nuget.org (i.e. the next RTM will include netstd), we should probably unify the build to produce only one set of packages and retire the "regular" packages. At that point we'd take out this temporary hack and let the traditional versioning approach do it's work.